src/main/java/competition/subsystems/turret/TurretSubsystem.java

```diff
@@ -44,20 +44,21 @@ public void turnRight() {
     }
 
     public void setPower(double power) {
-        if ((power < 0 && canTurnLeft()) || (power > 0 && canTurnRight())) {
+        if (canTurnLeft(power)) {
+            motor.simpleSet(power);
+        } else if (canTurnRight(power)) {
+            motor.simpleSet(power);
+        } else if (power == 0) {
             motor.simpleSet(power);
```

Comment:
This doesn't really make sense.
- If I can turn left, then anything is OK
- If I can turn right, the anything is OK
- If somebody says to use 0 power, then use 0 power.